### PR TITLE
dbeaver/dbeaver#21278 add left align to title and add spaces to title

### LINKED ISSUE #21278: Improve query readability on HTML export by aligning text to the left
**Is your feature request related to a problem? Please describe.**
Hey! As a data professional, I heavily use this "Copy as HTML" feature. (Thank you)
Ref.: [DataExporterHTML.java#L94](https://github.com/dbeaver/dbeaver/blob/e7989f5547716d31d1ec8d10247b9079173c730e/plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/stream/exporter/DataExporterHTML.java#L94)

When the query is exported along with the data, the text-align: center makes the query super unreadable. Changing that to "left" would already be a significant improvement.

I always have to edit this before sharing the query results with others. Sometimes, the tool I paste the results as HTML can't edit the alignment (e.g.: MS Teams), so I have to delete the query from the export and then copy and paste the query separately.

I know this is also used for the table name when the results are exported from a table. The table name would look better aligned to the left as well.

**Describe the solution you'd like**
Just change the alignment to: left
Bonus points: export the query as coloured source code using [HTMLSQLConverter.java#L38](https://github.com/dbeaver/dbeaver/blob/e7989f5547716d31d1ec8d10247b9079173c730e/plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/convert/impl/HTMLSQLConverter.java#L38)

**Describe alternatives you've considered**
I usually erase the query in the export, then copy and paste it again and share it outside the HTML table provided.

**Additional context**
Below are screenshots of the "Copy as HTML" when pasted to MS Teams. The query is only readable because it is simple. But the screenshots show that the query does not even appear unless you scroll to the center.
Snag_2e76ddd0
Snag_2e76f65a
Snag_2e770a7e

### PR DESCRIPTION
**Pull request by Destrolaric**
Closes dbeaver/dbeaver#21278
image

### CODE REVIEW

**Comment by ShadelessFox** [comment]
Wrong issue ID in the commit's message

File: bundles/org.jkiss.utils/src/org/jkiss/utils/CommonUtils.java [hunk 1]
```diff
@@ -933,11 +933,13 @@ public static String escapeHtml(@Nullable String text) {
         }
         return text
             .replace("&", "&amp;")
+            .replace(" ", "&nbsp")
```
Comment (ShadelessFox) [inline bundles/org.jkiss.utils/src/org/jkiss/utils/CommonUtils.java:936]: Is the semicolon optional here?

File: bundles/org.jkiss.utils/src/org/jkiss/utils/CommonUtils.java [hunk 2]
```diff
@@ -933,11 +933,13 @@ public static String escapeHtml(@Nullable String text) {
...
             .replace("<", "&lt;")
             .replace(">", "&gt;")
             .replace("\r\n", "<br>")
             .replace("\r", "<br>")
-            .replace("\n", "<br>");
+            .replace("\n", "<br>")
+            .replace("\t", "&emsp");
```
Comment (ShadelessFox) [inline bundles/org.jkiss.utils/src/org/jkiss/utils/CommonUtils.java]: And here

File: bundles/org.jkiss.utils/src/org/jkiss/utils/CommonUtils.java (hunk 1 above)
Comment (Destrolaric) [reply bundles/org.jkiss.utils/src/org/jkiss/utils/CommonUtils.java:936]: Works without semicolon, as shown at screenshot

Comment (ShadelessFox) [reply bundles/org.jkiss.utils/src/org/jkiss/utils/CommonUtils.java:936]: Although some named characters in HTML don't have the semicolon, the `&emsp;` is an exception.

Please see https://html.spec.whatwg.org/multipage/named-characters.html

**Review by ShadelessFox — APPROVED** [review]

**Comment by Matvey16** [comment]
Verified

**Pull request merged**